src/main/java/org/candlepin/controller/ContentAccessManager.java

```diff
@@ -715,9 +759,52 @@ else if (listUpdated) {
         return owner;
     }
 
-    private boolean isTurningOffSca(String updatedMode, String currentMode) {
-        return ContentAccessMode.ORG_ENVIRONMENT.matches(currentMode) &&
-            ContentAccessMode.ENTITLEMENT.matches(updatedMode);
+    /**
+     * Checks if the content access mode is transitioning and, if so, if it is transitioning to the
+     * target mode. That is, the current mode and updated modes are not equal, and the updated mode
+     * is equal to the target mode.
+     *
+     * @param current
+     *  the current content access mode name
+     *
+     * @param updated
+     *  the updated content access mode name
+     *
+     * @param target
+     *  the targeted content access mode to check
+     *
+     * @return
+     *  true if the content access mode is transitioning to the target mode; false otherwise
+     */
+    private boolean isTransitioningTo(String current, String updated, ContentAccessMode target) {
```

Comment:
This method doesn't appear to be used, just wanted to double check and see if you meant to include it. I can see that it could be helpful for future implementations where we need to check that we are updating to the target ContentAccessMode.

Reply:
Oh, yup, it ended up not being used. The first iteration of this revoked all entitlements on the changeover to SCA, but performance testing proved that to be non-viable in many cases so it got removed. The refactoring of the original method and the new companion method were left over as a result of that.

I'll remove the unused method but leave the other refactor in place.

Reply:
done